.env.example

```diff
@@ -0,0 +1,45 @@
+# Services
+CONVECTION_API_BASE=https://convection-staging.artsy.net/api
+DELTA_API_BASE=http://delta-staging.artsy.net
+DIFFUSION_API_BASE=https://diffusion-staging.artsy.net/api
+EMBEDLY_ENDPOINT=https://i.embed.ly/1/display
+GALAXY_API_BASE=https://artsy-galaxy-staging.herokuapp.com
+GEMINI_API_BASE=https://media.artsy.net
+GEMINI_ENDPOINT=https://d7hftxdivxxvm.cloudfront.net
+GOOGLE_CSE_API_BASE=https://www.googleapis.com/customsearch/v1
+GRAVITY_API_BASE=https://stagingapi.artsy.net/api/v1
+GRAVITY_API_URL=https://stagingapi.artsy.net/
+HMAC_SECRET=https://www.youtube.com/watch?v=DLzxrzFCyOs
+IMPULSE_API_BASE=https://impulse-staging.artsy.net/api
+POSITRON_API_BASE=https://writer-staging.artsy.net/api
+PREDICTION_ENDPOINT=https://live-staging.artsy.net
+REDIS_URL=redis://127.0.0.1:6379
+
+# Keys/Secrets
+CONVECTION_APP_ID=
+DIFFUSION_TOKEN=
+EMBEDLY_KEY=
+GALAXY_TOKEN=
+GOOGLE_CSE_CX=
+GOOGLE_CSE_KEY=
+GRAVITY_ID=
+GRAVITY_SECRET=
+IMPULSE_APPLICATION_ID=
+
+# Options
+BASIC_AUTH_USERNAME=foo
+BASIC_AUTH_PASSWORD=bar
```

Comment:
I suppose these were retrieved from prod/staging? Because these are no longer being used, assuming they were from when we still had basic auth in front of GraphiQL.

Reply:
These were retrieved from the `metaphysics-development.env` file in 1Password.  I did my best to concatenate everything I found there, in `.env.test` and on staging.  Good to know they are unused as this is not documented anywhere and the `.env.example` file is a first attempt to pull everything together.

Grep'ing around to `process.env` the used env vars are scattered all throughout the project.  It would be good to refactor this to pull in all environment to config in once place, for example in `./src/config.js` as a lot of this has been guesswork.

Reply:
@mzikherman @damassi When MP starts using TS, you can add type defs for known env variables so that MP code can’t silently use unknown env variables.